script/Ciel/Day 7/07_01_CIEL07_3.txt

```diff
@@ -472,6 +472,7 @@ If Arcueid is still here, then that must mean there are still vampires left to d
 // P: Roa is hard to classify, but yes, he's formally Rank VI because he hasn't been
 // P: allowed to go any further by the Ancestors. (Ancestor permission is required to ascend beyond Rank VI).
 // P: That said, I'm fairly certain this isn't referring to Roa but is rather actually a generic statement
+// same concern as earlier 下級死徒
```

Comment:
yeah, per comment, this is a generic statement rather than referring to the specific rank